.changeset/modern-chairs-sit.md

```diff
@@ -0,0 +1,17 @@
+---
+"@spectrum-css/dialog": major
+"@spectrum-css/modal": minor
```

Comment:
Do we agree with these version bumps? Is it a concern that the dialog component will have undergone 2 major version bumps between standard dialog, and then these 2 story updates? I think they're considered breaking changes.

Same for modal- is it a minor version bump?